backend/index.js

```diff
@@ -456,18 +457,52 @@ app.post('/api/uploadStory', upload.single('story'), async (req, res) => {
       data: {
         userId: parseInt(userId, 10),
         fileUrl: result.secure_url,
+        expiresAt: addHours(new Date(), 24), // Set expiration to 24 hours from now
```

Comment:
Nice, this is a good solution to the expiry logic now that we've shifted the complexity over to the stories-chaining